client/signup/main.jsx

```diff
@@ -45,7 +45,8 @@ import * as oauthToken from 'lib/oauth-token';
 /**
  * Constants
  */
-const MINIMUM_TIME_LOADING_SCREEN_IS_DISPLAYED = 8000;
+const MINIMUM_TIME_LOADING_SCREEN_IS_DISPLAYED_FREE = 8000;
+const MINIMUM_TIME_LOADING_SCREEN_IS_DISPLAYED_PREMIUM = 3000;
```

Comment:
Not sure if users will be able to read all the text (mainly the email confirmation text) in three seconds. I would give it at least 4-5 seconds. Also, is the "waiting page" fully translated? If not, we should definitely increase the wait time as non-native speakers need more time for reading English. :)